app/uk/gov/hmrc/merchandiseinbaggagefrontend/views/SearchGoodsCountryView.scala.html

```diff
@@ -0,0 +1,75 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.merchandiseinbaggagefrontend.views.html.components
+@import uk.gov.hmrc.merchandiseinbaggagefrontend.controllers.routes._
+
+@this(
+layout: Layout,
+formHelper: FormWithCSRF,
+h1: components.h1,
+errorSummary: components.errorSummary,
+button: components.button
+)
+
+@(form: Form[_])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@head = {
+    <link rel="stylesheet" href="@routes.Assets.versioned("stylesheets/accessible-autocomplete.min.css")" >
+}
+
+@scripts = {
+    <script src="@routes.Assets.versioned("javascripts/accessible-autocomplete.min.js")"></script>
```

Comment:
Will this page work if the user has javascript disabled?

Reply:
Good question, I expect not, but do we need to cater for that? I think this is a question for Content Designer

Reply:
Yes - will discuss on Slack.

Reply:
Sarthak will test with and without. We think this is safe (defaults to a select).